src/analysis/retail/evoker/preservation/modules/talents/ConsumeFlame.tsx

```diff
@@ -121,7 +111,7 @@ class ConsumeFlame extends Analyzer {
             </li>
             <li>
               4 stacks of <SpellLink spell={TALENTS_EVOKER.TEMPORAL_COMPRESSION_TALENT} /> if you
-              have <ItemSetLink id={EVOKER_TWW1_ID} /> 4 Piece
+              have the <SpellLink spell={TALENTS_EVOKER.TEMPO_CHARGED_TALENT} /> talent
```

Comment:
```suggestion
              have <SpellLink spell={TALENTS_EVOKER.TEMPO_CHARGED_TALENT} /> talented
```

nit: use same phrasing as previous bullet point